moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -657,17 +661,18 @@ class LockedPlanningSceneRO
 
   operator bool() const
   {
-    return planning_scene_monitor_ && planning_scene_monitor_->getPlanningScene();
+    planning_scene_monitor::LockedPlanningSceneRO ls(planning_scene_monitor_);
+    return planning_scene_monitor_ && ls;
```

Comment:
Why would you change the behavior of this function and did you try running this? Before, it would check if psm and planning scene are valid (!=nullptr). Now it seems to recursively check `LockedPlanningSceneRO::bool()`.